osu.Game.Rulesets.Mania/UI/Components/ColumnHitObjectArea.cs

```diff
@@ -78,21 +60,86 @@ public Color4 AccentColour
 
                 accentColour = value;
 
-                updateColours();
+                if (hitTarget is IHasAccentColour colouredHitTarget)
+                    colouredHitTarget.AccentColour = accentColour;
             }
         }
 
-        private void updateColours()
+        private class DefaultHitTarget : CompositeDrawable, IHasAccentColour
         {
-            if (!IsLoaded)
-                return;
+            private readonly IBindable<ScrollingDirection> direction = new Bindable<ScrollingDirection>();
 
-            hitTargetLine.EdgeEffect = new EdgeEffectParameters
+            private readonly Container hitTargetLine;
+            private readonly Drawable hitTargetBar;
+
+            public DefaultHitTarget()
             {
-                Type = EdgeEffectType.Glow,
-                Radius = 5,
-                Colour = accentColour.Opacity(0.5f),
-            };
+                InternalChildren = new[]
+                {
+                    hitTargetBar = new Box
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        Height = NotePiece.NOTE_HEIGHT,
+                        Alpha = 0.6f,
+                        Colour = Color4.Black
+                    },
+                    hitTargetLine = new Container
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        Height = hit_target_bar_height,
+                        Masking = true,
+                        Child = new Box { RelativeSizeAxes = Axes.Both }
+                    },
+                };
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(IScrollingInfo scrollingInfo)
+            {
+                direction.BindTo(scrollingInfo.Direction);
+                direction.BindValueChanged(dir =>
+                {
+                    Anchor anchor = dir.NewValue == ScrollingDirection.Up ? Anchor.TopLeft : Anchor.BottomLeft;
+
+                    hitTargetBar.Anchor = hitTargetBar.Origin = anchor;
```

Comment:
not sure doing the full bindable flow here is required (could just pass on the anchor from self to children) but should be fine for now.